packages/helpers/src/tests/handle-initial-channel-messages.spec.ts

```diff
@@ -41,7 +41,7 @@ describe('handle initial channel creation', () => {
     // get the blue RFI channel
     const rfiChan = res.channels['channel-BlueRFI']
     expect(rfiChan).toBeTruthy()
```

Comment:
thanks for the fix @lilitkarapetyan 

_One day_ I look forward to `handle-initial-channel-messages` getting refactored & tidied up